apps/ledger-live-mobile/src/families/osmosis/shared/ValidatorRow.tsx

```diff
@@ -0,0 +1,114 @@
+import { getAccountUnit } from "@ledgerhq/live-common/account/index";
+import { CosmosValidatorItem } from "@ledgerhq/live-common/families/cosmos/types";
+import { LEDGER_OSMOSIS_VALIDATOR_ADDRESS } from "@ledgerhq/live-common/families/osmosis/utils";
+import { AccountLike } from "@ledgerhq/live-common/types/index";
+import { Text } from "@ledgerhq/native-ui";
+import React, { useCallback } from "react";
+import { Trans } from "react-i18next";
+import { StyleSheet, View } from "react-native";
+import CurrencyUnitValue from "../../../components/CurrencyUnitValue";
+import Touchable from "../../../components/Touchable";
+import ValidatorImage from "../../cosmos/shared/ValidatorImage";
+
+const ValidatorRow = ({
+  onPress,
+  validator,
+  account,
+}: {
+  onPress: (v: CosmosValidatorItem) => void;
+  validator: CosmosValidatorItem;
+  account: AccountLike;
+}) => {
+  const onPressT = useCallback(() => {
+    onPress(validator);
+  }, [validator, onPress]);
+
+  return (
+    <Touchable
+      event="DelegationFlowChosevalidator"
+      eventProperties={{
+        validatorName: validator.name || validator.validatorAddress,
+      }}
+      onPress={onPressT}
+    >
+      <View style={styles.validator}>
+        <ValidatorImage
+          isLedger={
+            LEDGER_OSMOSIS_VALIDATOR_ADDRESS === validator.validatorAddress
+          }
+          size={32}
+          name={validator.name ?? validator.validatorAddress}
+        />
+        <View style={styles.validatorBody}>
+          <Text
+            numberOfLines={1}
+            fontWeight="semiBold"
+            style={styles.validatorName}
+          >
+            {validator.name || validator.validatorAddress}
+          </Text>
+          {true ? (
+            <Text
+              fontWeight="semiBold"
+              numberOfLines={1}
+              style={styles.overdelegated}
+            >
+              <Trans i18nKey="cosmos.delegation.commission" />{" "}
+              {validator.commission * 100} %
```

Comment:
it would make sense to approximate the percentage. like `toFixed(2)` OR `Math.round(validator.commission * 100 * p) / p` where p is one of 10, 100, 1000,...

Reply:
Hey @gre, we based this code off the existing implementation in Cosmos - if toFixed or Math.round work better we should consider updating the Cosmos implementation as well.